instructors.md

```diff
@@ -3,28 +3,126 @@ layout: page
 title: Version Control with Git
 subtitle: Instructor's Guide
 ---
-## Legend
+Version control might be the most important topic we teach,
+but Git is definitely the most complicated tool.
+However,
+GitHub presently dominates the open software forge landscape,
+so we have to help novices learn just enough Git
+to feel they can and should learn more on their own.
 
-Wolfman and Dracula will work together in one project.
+This is why we don't teach branching:
```

Comment:
There are excellent branching tutorials and simulations available on-line. If you teach branching, you run a high likelihood of confusing those new to version control and/or command line use. Also branching adds complexity since there are various workflows that can be used in practice. Better to make it an intermediate lesson or advanced lesson (even if self directed for those that already know git).